spec/services/scrape_onet_codes_spec.rb

```diff
@@ -34,10 +34,33 @@
       expect(onet3.code).to eq "17-2011.00"
       expect(onet3.version).to eq "2019"
     end
+
+    it "creates onet records with extra whitespaces" do
```

Comment:
it looks like the test expectations are for the white space to be removed. I think the test name could be edited to reflect that.